README.md

```
@@ -1,8 +1,8 @@
-# evscc
+# oxic
 
-![](https://github.com/septechx/evsc/actions/workflows/test.yml/badge.svg)
+![](https://github.com/septechx/oxi/actions/workflows/test.yml/badge.svg)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)